spec.html

```diff
@@ -38578,7 +38579,7 @@ <h1>
           <emu-alg>
             1. Assert: _p_ is a canonical, unaliased Unicode property name listed in the “Canonical property name” column of <emu-xref href="#table-nonbinary-unicode-properties"></emu-xref>.
             1. Assert: _v_ is a property value or property value alias for the Unicode property _p_ listed in <a href="https://unicode.org/Public/UCD/latest/ucd/PropertyValueAliases.txt"><code>PropertyValueAliases.txt</code></a>.
-            1. Let _value_ be the canonical property value of _v_ as given in the “Canonical property value” column of the corresponding row.
+            1. Let _value_ be the value in the “long name” field of the corresponding line in that file.
             1. Return the List of Unicode code points _value_.
```

Comment:
Why does this AO return "a Unicode property value", but here "Return the List of ..."?

Reply:
Both UnicodeMatchProperty and UnicodeMatchPropertyValue are implicitly implementing each of "Unicode property name" and "Unicode property value" as a List of code points (similar to [ECMAScript source text](https://tc39.es/ecma262/multipage/ecmascript-language-source-code.html#sec-source-text) and the output of [StringToCodePoints](https://tc39.es/ecma262/multipage/ecmascript-language-source-code.html#sec-stringtocodepoints)), which I think is necessary for comparison against values in the [Unicode property tables](https://tc39.es/ecma262/multipage/text-processing.html#table-nonbinary-unicode-properties). There may be a better way, but this is is valid AFAICT.

Reply:
I don't think it's invalid, but I also don't think it needs to be made explicit. IMO it can be returned directly. Though that should probably be a separate PR.

Reply:
Actually I was pretty unhappy with the wording used in this PR to refer to external data or table values. I've pushed a commit to make it a little more consistent and explicit. This has also inspired me to add two new editorial conventions:

* assertion steps should not be necessary to understand an algorithm; a reader should interpret it the same if they are removed
* do not use determiners or pronouns to refer to values from other steps (e.g. "that number"); instead uses aliases or repetition